docs/basics/chat_interface.md

```diff
@@ -0,0 +1,131 @@
+# Chat Interface
```

Comment:
Please expose this in documentation (probably linking through basics/inference under a new section) as well as in https://github.com/NVIDIA/NeMo-Skills/blob/main/mkdocs.yml